oncokb/oncokbETL.py

```diff
@@ -196,11 +234,19 @@ def transform(df, engine):
     df['Chromosome'] = df['Chromosome'].apply(chr_change).copy()
     df.columns = db_cols
     df['CELLLINENAME'] = df['CELLLINENAME'].apply(cell_line_changes).copy()
-    df['HUGOGENESYMBOLSEQNBR'] = df['HUGOGENESYMBOLSEQNBR'].map(hugo_dict)
+    df['HUGOGENESYMBOLSEQNBR'] = df['HUGOGENESYMBOLSEQNBR'].map(hugo_dict).copy()
+    # Remove all rows that have no HUGO Gene mapping
+    df = df[df['HUGOGENESYMBOLSEQNBR'].notna()]
```

Comment:
Fantastic code! I noticed one minor discrepancy: I compared the oncoKB data that I have uploaded, my data set has 74069 rows of oncoKB data, while your data set has 3871 rows of oncoKB data. After digging into it, I found that I mapped to value 0 for Hugogenesymbolseqnbr column when the value of hugo_symbol does not match to the other values in common.hugogenesymbol table. Maybe we can discuss this requirement with DI? What do you think? 

Common.Hugogenesymbol table:
0, None Found

Reply:
Right-- so essentially if it does not have a Hugo Gene Symbol, we can not insert it into Oracle because it needs that number-- a non-nullable field.

The one lookup table, HugoGeneSymbol, has a set of all the various genes that these mutations could stem from.  Definitely something we can mention for discussion.  That is the only "filter" we have as I have to have it or else it won't insert into OncoKBGenePanel table.